js/src/features/human-resouce/actions.ts

```diff
@@ -158,8 +158,22 @@ export const closeAbsenceUpdateModal = createAction(
   "CLOSE_ABSENCE_UPDATE_MODAL",
 )();
 
-export const openAbsenceCreateForm = createAction("OPEN_ABSENCE_CREATE_FORM")();
+export const openAbsenceCreateModal = createAction(
```

Comment:
Why did you rename openAbsenceCreateForm to openAbsenceCreateModal? If you want to make the form non-modal, you will have to rename the action

Reply:
I renamed it because before form was inline and now it's in modal. I think it's ok. I have in state  `isCreateModalVisible` and it is a corresponding action.